src/client/event_handler.rs

```diff
@@ -21,6 +21,7 @@ macro_rules! event_handler {
                 $( #[deprecated = $deprecated] )?
                 async fn $method_name(&self, $($context: Context,)? $( $arg_name: $arg_type ),*) {
                     // Suppress unused argument warnings
+                    #[allow(clippy::double_parens)]
                     drop(( $($context,)? $($arg_name),* ))
```

Comment:
```suggestion
                    drop(( $($context,)? $($arg_name,)* ))
```
This way the macro will produce 1-tuples instead of parenthesized single values.